pkg/activator/handler/main_test.go

```diff
@@ -69,7 +69,7 @@ func BenchmarkHandlerChain(b *testing.B) {
 	})
 
 	// Make sure to update this if the activator's main file changes.
-	ah := New(ctx, fakeThrottler{}, rt, false, logger)
+	ah := New(ctx, fakeThrottler{}, rt, false, logger, false /* TLS */)
```

Comment:
what is the overhead if tls is `true`, does it make sense to have run a benchmark with that too?